### ISSUE TITLE
[REQUEST] Proxy

### ISSUE THREAD

**Issue by saalexeev** [kokarare1212/librespot-python#176]
**Is your feature request related to a problem? Please describe.**
I use multiple spotify accounts to upload tracks. I would like to be able to use a proxy server for each account, how can this be achieved?

**Describe the solution you'd like**
There is proxy options commented in session configuration. I'd like to know how can I use them

**Comment by kokarare1212** [comment]
Yes, the current proxy implementation is lame and commented out.
Specifically, we have most socket proxies and proxies for some APIs that use HTTP to implement proxies under the hood, but we don't implement socket proxies.
https://github.com/kokarare1212/librespot-python/issues/160#issuecomment-1368577229

**Comment by van4oza** [comment]
Any plans on proxies yet? mb they can be implemented system-wide with env vars like HTTP_PROXY for requests?
